systolic/translate/CgraMemRightAndBottomRTL_matmul_2x2_test.py

```diff
@@ -80,6 +80,7 @@ def check_parity(s):
     return True
 
   def line_trace(s):
+    # return s.dut.verbose_trace(verbosity = 2)
```

Comment:
Enable by default?

Reply:
There're both [`-xvs --tb=short`](https://github.com/tancheng/VectorCGRA/blob/49a40187f90a8d67b0a410c0d3bc5944422e22e6/.github/workflows/python-package.yml#L72) and [`--tb=short --test-verilog --dump-vtb --dump-vcd`](https://github.com/tancheng/VectorCGRA/blob/49a40187f90a8d67b0a410c0d3bc5944422e22e6/.github/workflows/python-package.yml#L73C87-L73C134) in testcases, if enable verbose_trace by default, can I remove and ignore `--tb=short --test-verilog --dump-vtb --dump-vcd` case in action workflow?